src/lolex-src.js

```diff
@@ -544,6 +544,7 @@ function withGlobal(_global) {
             });
         };
         clock.setInterval = function setInterval(func, timeout) {
+            timeout = parseInt(timeout, 10);
```

Comment:
Is the radix correct?

Looking at the spec - I think `Number` is more correct here. It's really a tiny nit, but if someone passes  `0x100` to setInterval or setTimeout the correct timeout according to the timer spec is 255 and not 0 (which this produces).

Not a big deal by any measure.

Reply:
True, the parse function assumes to get a decimal number (10). I think we could leave this off to let the function decide which radix to use depending on the numerical system.

What do you mean with
> I think `Number` is more correct here

Maybe to use the `Number.parseInt()` function? This should be the same as the global one afaik.